xl_auth/templates/users/inspect.html

```diff
@@ -0,0 +1,169 @@
+<!-- users/inspect.html -->
+{% extends "layout.html" %}
+{% block content %}
+    <h1>{{ _('Inspect User \'%(email)s\'', email=user.email) }}</h1>
+    <br/>
+    <div class="panel panel-default">
+        <dl class="dl-horizontal">
+            <dt>{{ _('Last Login At') }}:</dt>
+            <dd>{{ '-' if not user.last_login_at else user.last_login_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}</dd>
+
+            <dt>{{ _('Active') }}:</dt>
+            <dd>{{ _('Yes') if user.is_active else _('No') }}</dd>
+
+            <dt>{{ _('Administrator') }}:</dt>
+            <dd>{{ _('Yes') if user.is_admin else _('No') }}</dd>
+
+            <dt>{{ _('Cataloguing Administrator') }}:</dt>
+            <dd>{{ _('Yes') if user.is_cataloging_admin else _('No') }}</dd>
+
+            <dt>{{ _('Users Created') }}:</dt>
+            <dd>{{ users_created }}</dd>
+
+            <dt>{{ _('Users Modified') }}:</dt>
+            <dd>{{ users_modified }}</dd>
+
+            <dt>{{ _('Permissions Created') }}:</dt>
+            <dd>{{ permissions_created }}</dd>
+
+            <dt>{{ _('Permissions Modified') }}:</dt>
+            <dd>{{ permissions_modified }}</dd>
+
+            <dt>{{ _('Collections Created') }}:</dt>
+            <dd>{{ collections_created }}</dd>
+
+            <dt>{{ _('Collections Modified') }}:</dt>
+            <dd>{{ collections_modified }}</dd>
+
+            <dt>{{ _('Clients Created') }}:</dt>
+            <dd>{{ clients_created }}</dd>
+
+            <dt>{{ _('Clients Modified') }}:</dt>
+            <dd>{{ clients_modified }}</dd>
+
+            <dt>{{ _('Last Modified') }}:</dt>
+            <dd>
+                {{ user.modified_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.inspect', user_id=user.modified_by.id) }}">
+                    {{ user.created_by.full_name }}
```

Comment:
Should be `user.modified_by.full_name` here?

Reply:
Correct!!